standards/problem-solving.md

```diff
@@ -0,0 +1,24 @@
+
+- Psuedocode is used to break down problems by using steps to find the solution. It does not require actual syntax to be written out, and it is human readable which means it does not get compiled by the computer. Devs can write the steps on notepad, whiteboard, or even make temporary comments in a IDE. The point is to breakdown a huge problem into chucks and connect the dots to find a solution in an efficient way. Once the steps seem logical, devs will attempt to convert this psuedocode into code and test their logic. Read more about this here [What is Pseudocode? How to Use Pseudocode to Solve Coding Problems](https://www.freecodecamp.org/news/what-is-pseudocode-in-programming/)
+
+
+## Find Resources 
+It is impossible to retain every thing learned while being a developer. Google is your best friend. The amount of resources available to help complete task is vast. For Example, [MDN](https://developer.mozilla.org/en-US/), [CSS-Tricks](https://css-tricks.com/), [Javascript.info](https://javascript.info/), are a few resources for front-end development. Reading Documentation on how a certain library work is key when using it on a project. The documentation should always be a reference when using it. Search for your problem using the error message keywords given to you. Other devs have probably ran into the same error as you and may have solved it. How you google is just as important as what you google, so click here to learn. [11 Tricks To Master the Art of Googling as a Software Developer](https://betterprogramming.pub/11-tricks-to-master-the-art-of-googling-as-a-software-developer-2e00b7568b7d)  
+
+## Problem solving without coding
+Problem solving goes beyond just coding. We use it in our daily life all the time. One of the best ways to become a better problem solver is by making it personal. **Video games** require lots of problem solving. Think about it, the game gives the player a challenge, and the player has to find ways to beat the challenge or risk not going to the next level. **Puzzles** are another good problem solving exercise to find the shape that fits other parts of the puzzle. **Cooking** is another form of problem solving. To follow a recipe, people have gather the items, mix the right amount of spices or seasonings to have the desired meal. As a dev, it is vital to find a hobby that involves problem solving. Read more here about what the greats like Mark Zuckerberg and Elon Musk use to help solve problems. [https://www.freecodecamp.org/news/how-to-think-like-a-programmer-lessons-in-problem-solving-d1d8bf1de7d2/](https://www.freecodecamp.org/news/how-to-think-like-a-programmer-lessons-in-problem-solving-d1d8bf1de7d2/) 
+
```

Comment:
Suggestion: Ask other developers for help. Do pair programming. Talk to other developer before talking to rubber ducky.

Reply:
Yeah, I see what you are saying. So, the idea behind this page is to try to help developers  become self sufficient before reaching out for help. I know there is already a dedicated steps for asking and this is why I did not mention asking for help in this page to avoid repetition. Thanks you for the feedback

Reply:
Got it. Awesome page. Great job!